src/transformers/models/fuyu/processing_fuyu.py

```diff
@@ -621,12 +630,14 @@ def __call__(
             # images -> List[PIL.Image.Image]
 
             if self.image_placeholder_tag not in current_text: # No image tags
-                interleaved.append([[current_text, None]])
+                interleaved.append([[None, current_text]])
```

Comment:
It seems that you are still implementing image-text pairs and then concatenate. Why do we have to do it? Can we have a simpler design: just input image patches to the positions of <image>?